script.js

```
@@ -0,0 +1,402 @@
+    // 更新显示
+    updateDisplay() {
+        this.display.current.textContent = this.formatNumber(this.currentInput);
+        this.display.history.textContent = this.history;
+    }
+    
+    // 格式化数字显示
+    formatNumber(num) {
+        if (num === 'Error') return num;
+        
+        const number = parseFloat(num);
+        if (isNaN(number)) return num;
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)